src/routes/promises/[id]/+page.svelte

```diff
@@ -67,6 +68,14 @@
 			ดูความหมายสถานะอื่นๆ
 		</button>
 	</div>
+	{#if promise.clarificationLogs && promise.clarificationLogs?.length > 0}
```

Comment:
Should be the last one: Maybe you don't need `&&` since `promise.clarificationLogs?.length > 0` is already included the first condition with optional chaining (`?`).

Reply:
i think ts compiler is not smart enough to detect that `promise.clarificationLogs` cannot be `undefined` without the first clause?

<img width="874" alt="image" src="https://github.com/user-attachments/assets/30c4c76b-7b36-471f-9183-b3bea3c46151">

Reply:
OMG THAT'S SO SAD

Well, I think I'll merge it now. Thank you for your hard work! Will reach back to you at the end of the campaign for the sticker!